android/app/src/main/java/network/mysterium/service/core/NodeRepository.kt

```diff
@@ -129,7 +133,13 @@ class NodeRepository(private val deferredNode: DeferredNode) {
         req.providerID = providerID
         req.serviceType = serviceType
         req.identityAddress = identityAddress
-        deferredNode.await().connect(req)
+        val res = deferredNode.await().connect(req) ?: return@withContext
```

Comment:
@anjmao MobileNode.class#public native void connect(ConnectRequest var1) throws Exception;
returns void.
Where is this res coming from? I get build error, as there's no res.errorCode

Reply:
Node 0.22.1